### PR TITLE
implemented both functions, not sure if this is correct but output se…

### PR DESCRIPTION
**Pull request by tgentry300**
…ems correct

### CODE REVIEW

File: mimic.py [hunk 1]
```diff
@@ -45,18 +45,31 @@
...
 
 
 def mimic_dict(filename):
     """Returns mimic dict mapping each word to list of words which follow it."""
-    # +++your code here+++
-    raise NotImplementedError("Get to Work!")
+    list_of_words = read_file(filename)
+    mimic_dict = {}
+    for i, word in enumerate(list_of_words):
+        mimic_dict[word] = list_of_words[i + 1:]
```
Comment (jakeberg) [inline mimic.py]: This is giving you a dictionary of words that have every word that comes after it in the string. What you want is a dictionary with a key that is a word and the value is every word that come IMMEDIATELY  after that word.

File: mimic.py [hunk 2]
```diff
@@ -45,18 +45,31 @@
...
+
 
 
-def print_mimic(mimic_dict, word):
-    """Given mimic dict and start word, prints 200 random words."""
-    # +++your code here+++
-    raise NotImplementedError("Get to Work!")
+def print_mimic(mimic_dict):
+    """Given mimic dict and start word, prints 200 random words. Removed second argument as it is not needed for my implementation"""
+    if len(mimic_dict) >= 200:
```
Comment (jakeberg) [inline mimic.py]: This is always going to be true. I think you want to be appending the word to a string and be checking the length of that string.

File: mimic.py [hunk 3]
```diff
@@ -45,18 +45,31 @@
...
+        print random.sample(mimic_dict.items(), 200)
```
Comment (jakeberg) [inline mimic.py]: Try this in a repl.it. By using .items() you are just getting a tuple of the key and value in the dictionary.

File: mimic.py (hunk 1 above)
Comment (tgentry300) [reply mimic.py]: Yeah, Piero and I talked about it already, just haven't reimplemented it yet, Thanks for the constructive comments though.

**Pull request closed without merging**